docs/about/features_index/fed_eval.rst

```diff
@@ -28,9 +28,9 @@ The following steps can be leveraged to achieve practical e2e usage of FedEval
 
 *N.B*: We will be using torch/mnist plan itself for both training and with some minor changes for evaluation as well
 
-*Prerequisites*: Please ensure that OpenFL version==1.7 is installed or you can also choose to install latest from source.
+*Prerequisites*: Please ensure that OpenFL version>=1.8 is installed or you can also choose to install latest from source.
 
-With OpenFL version==1.7 aggregator start command is enhanced to have an optional argument '--task_group' which, as the help suggest, will select the provided task_groups task to assigner for execution in the collaborator(s), since this defaults to 'learning'
+Since OpenFL 1.7 release aggregator start command is enhanced to have an optional argument '--task_group' which, as the help suggest, will select the provided task to assigner for execution in the collaborator(s)

```

Comment:
I don't think mentioning 1.7 is useful here (to not confuse a user more than the fact that 1.8 is mentioned)